README.md

```diff
@@ -279,7 +283,20 @@ const logger = createLogger({
 })
 ```
 
-Complete samples for Express, Fastify and Koa are available in `/samples/` directory.
+### Pino
+
+This is how you can integrate `cls-rtracer` with [pino](https://github.com/pinojs/pino) logger.
+
+```javascript
+
+// mixin function adds properties of the returned object to the logged JSON.
+const logger = require('pino')({
+    mixin () {
+        return { request_id: rTracer.id() }
```

Comment:
I'd prefer to use camel case for field names. It feels more natural to do so, considering that Pino is positioned as a JSON logger:
```suggestion
        return { requestId: rTracer.id() }
```

Reply:
Done